UPGRADING.rst

```diff
@@ -4,6 +4,12 @@ Upgrading to Graylog 4.0.x
 
 .. _upgrade-from-33-to-40:
 
+Mark 'List of streams' in Streams API als deprecated
+====================================================
+
+In 4.0 we introduce a new API endpoint to retrieve streams from the backend: '/streams/paginated'.
+We therefore mark '/streams' as deprecated. This endpoint will be removed in the next version.
```

Comment:
It would be helpful to a) mention that `/streams` will continue to exist, but will have the request & response format of `/streams/paginated` and b) advise the user to start preparing any consumer of the `/streams` format to accept the new response format so only the endpoint URL needs to be switched for the next version.